src/Backend/Modules/Slideshows/Engine/Model.php

```diff
@@ -23,6 +24,8 @@ class Model
 
     const IMAGE_FOLDER = '/Slideshows/';
 
+    const MODULE = 'Slideshows';
```

Comment:
why did you move this to a constant? it is the Slideshows module, I don't really see what the use is of having it is a constant